index.js

```diff
@@ -0,0 +1,15 @@
+const dirTree = require('directory-tree')
```

Comment:
`;` at the end of lines.

When there are multiple `const` lines like this, can have `,` at the end of the line, with `;` at the end of the final line.